### PR TITLE
Update OSDF SLAs for FabAID

### PR DESCRIPTION
**Pull request by brianhlin**
Plus bonus commits for minor updates to the general SLA doc

### CODE REVIEW

File: docs/SLA/osdf-origin.md [hunk 1]
```diff
@@ -19,13 +20,14 @@ General Service Level Agreement
...
+When clients use SciTokens to interact with objects made available through an Origin, the bearer token is passed to the
+Origin from an OSDF Cache.
 
 Service Availability
 --------------------
 
 #### Availability Definition
 
-Can successfully read a test file
+Web engine and federation API health endpoints report as healthy
```
Comment (brianhlin) [inline docs/SLA/osdf-origin.md:31]: It would be nice to use the director or self tests here but we don't necessarily control the backing store so there are fuzzy operational boundary conditions.

File: docs/SLA/osdf-cache.md [hunk 2]
```diff
@@ -19,13 +22,15 @@ General Service Level Agreement
...
+When clients use SciTokens to authenticate with the Cache, the bearer token is passed to the Cache
+(which may subsequently use it to impersonate the client to communicate with the origin).
 
 Service Availability
 --------------------
 
 #### Availability Definition
 
-Can successfully read a test file through the Cache server
+-   Web engine, director, and federation health API endpoints report as healthy
```
Comment (brianhlin) [inline docs/SLA/osdf-cache.md:33]: We may also want to hit the xrootd component API endpoint since that corresponds to the self-test. But I don't see a case where the director test succeeds and the self test fails.

File: docs/SLA/osdf-origin.md [hunk 3]
```diff
@@ -19,13 +20,14 @@ General Service Level Agreement
 Security Considerations
 -----------------------
 
-When clients use SciTokens to authenticate with the origin, the bearer token is passed to the origin from the cache
+When clients use SciTokens to interact with objects made available through an origin, the bearer token is passed to the
+Origin from an OSDF Cache.
```
[the remark this answers is not in the document]
Comment (brianhlin) [reply docs/SLA/osdf-origin.md]: Service capitalization is consistent with other SLA docs
